File: SEM5_CODE/PYTHON/BASIC/05_add_numbers.py
first_number = float(input("Enter first number: "))    # Read the first number
second_number = float(input("Enter second number: "))  # Read the second number

addition = first_number + second_number                # Add the numbers
subtraction = first_number - second_number             # Subtract the numbers
multiplication = first_number * second_number          # Multiply the numbers

print("Addition:", addition)                           # Display the addition
print("Subtraction:", subtraction)                     # Display the subtraction
print("Multiplication:", multiplication)               # Display the multiplication
if second_number != 0:                                 # Check before dividing
    division = first_number / second_number            # Divide the numbers
    print("Division:", division)                       # Display the division
else:
    print("Division is not possible by zero")          # Display the error

File: SEM3_CODE/PHP/BASIC/05_add_numbers.php
<?php
$firstNumber = (float) readline('Enter first number: ');    // Read the first number
$secondNumber = (float) readline('Enter second number: ');  // Read the second number

$addition = $firstNumber + $secondNumber;                   // Add the numbers
$subtraction = $firstNumber - $secondNumber;                // Subtract the numbers
$multiplication = $firstNumber * $secondNumber;             // Multiply the numbers

echo 'Addition: ' . $addition . PHP_EOL;                    // Display the addition
echo 'Subtraction: ' . $subtraction . PHP_EOL;              // Display the subtraction
echo 'Multiplication: ' . $multiplication . PHP_EOL;        // Display the multiplication

if ($secondNumber != 0) {                                   // Check before dividing
    $division = $firstNumber / $secondNumber;               // Divide the numbers
    echo 'Division: ' . $division;                          // Display the division
} else {
    echo 'Division is not possible by zero';                // Display the error
}
?>
